src/plugins/integrations/AWSIntegration.ts

```diff
@@ -209,10 +212,11 @@ class AWSIntegration implements Integration {
               activeSpan.setTag(SpanTags.TOPOLOGY_VERTEX, true);
               activeSpan.setTag(SpanTags.TRIGGER_DOMAIN_NAME, LAMBDA_APPLICATION_DOMAIN_NAME);
               activeSpan.setTag(SpanTags.TRIGGER_CLASS_NAME, LAMBDA_APPLICATION_CLASS_NAME);
+              activeSpan.setTag(AwsS3Tags.OBJECT_NAME, request.params.Key);
```

Comment:
What if `request.params.Key` is undefined? I mean, there is no `Key` in the request

Reply:
Then `AwsS3Tags.OBJECT_NAME` will be `undefined` and it will not be serialized during reporting.